### DISCUSSION TITLE
How to Open Two Instances of Artisan Without Starting Artisan Viewer?

### DISCUSSION THREAD

**Discussion by JoaoLuisC** [artisan-roaster-scope/artisan#1611] [Q&A] [answered]
Hello everyone,

I am working with Artisan version 1.8.2 and need to open two instances of the application simultaneously to monitor different roasting profiles at the same time. However, whenever I try to open the Artisan executable a second time, it starts Artisan Viewer, which doesn't allow me to see the temperatures and other necessary data.

Does anyone know how I can open two full instances of Artisan without the second one turning into Artisan Viewer?

Thank you in advance for your help!

**Comment by poundy** [comment, marked as answer]
you cannot open a second instance in "full" mode on the same computer. Artisan is programmed to open the second instance in viewer mode explicitly. 

Viewer allows access to the temperature data though. I am not sure how you use Artisan, but you can use the cursor and the time and temperature axis values are displayed, and you can still open Roast Properties and the Data tab has all the data that was recorded, so it seems unnecessary to have anything more??

**Reply by JoaoLuisC** [reply]
In my project, I need to monitor the temperature of 4 roasters at the same time, and to save money, I think the easiest way to monitor all 4 roasters would be with a virtual machine. Later on, if my professors approve, I will share the project with you so you can understand what I am trying to do. But thank you very much.

**Reply by poundy** [reply]
oh right, "see temperatures" really meant "read temperature probes". Yes, with multi-barrel setups, you need multiple physical or virtual machines to run separate instances.
